### PR TITLE
Pass ENVIRONMENT variables as test runner arguments.

### PR DESCRIPTION
**Pull request by awels**
- Added the ability to set environment variables to override arguments
  for the e2e test runner.

Signed-off-by: Alexander Wels <awels@redhat.com>

### CODE REVIEW

**Comment by awels** [comment]
@jeffvance @copejon This should allows you to set arguments to pass to the test runner so make functest will work for your environment as well.

File: hack/build/functests.sh [hunk 1]
```diff
@@ -22,14 +22,25 @@ set -e
 source hack/build/common.sh
 source hack/build/config.sh
 
-kubectl=${CDI_DIR}/cluster/.kubectl
+# collect all arguments to pass to test runner
+installed_namespace=${CDI_INSTALL_NAMESPACE:-"kube-system"}
+
+if [[ -z "${KUBECTL_PATH}" ]]; then
```
Comment (jeffvance) [inline hack/build/functests.sh]: I think you can do this in a single assignment. Something like:
```
kubectl="${KUBECTL_PATH:-${CDI_DIR}/cluster/.kubectl}"
```

**Comment by jeffvance** [comment]
@awels does this pr mean that I should not default any of the flags in my framework pr? Not sure if there's a case where default the flags in `init` might still be useful.

**Comment by awels** [comment]
Good point, but if you run outside of the script you will still need
defaults.

File: hack/README.md [hunk 2]
```diff
@@ -73,10 +73,14 @@ These may be passed to a target as `$ make VARIABLE=value target`
 #### Execute Functional Tests
 Environment Variables and Supported Values
 
-| Env Variable       | Default       | Additional Values  |
-|--------------------|---------------|--------------------|
-|KUBEVIRT_PROVIDER   | k8s-1.10.4    | os-3.10.0          |
-|NUM_NODES           | 1             | 2-5                |
+| Env Variable       | Default       | Additional Values       |
```
Comment (jeffvance) [inline hack/README.md]: nice!

**Comment by awels** [comment]
@jeffvance @copejon I believe everything in this patch is good to go.

**Comment by jeffvance** [comment]
lgtm

**Comment by jeffvance** [comment]
@awels @copejon with 353 merged can we close this?

**Comment by awels** [comment]
Closing as #353 has the implemented functionality.

**Pull request closed without merging**